core/state/plain_state_reader.go

```diff
@@ -78,3 +79,14 @@ func (r *PlainStateReader) ReadAccountIncarnation(address common.Address) (uint6
 	}
 	return binary.BigEndian.Uint64(b), nil
 }
+
+func (r *PlainStateReader) ReadVersion() (uint64, error) {
```

Comment:
This does not need to be wrapped into `PlainStateReader`, instead it can go to the `core/rawdb` package perhaps? Looking at the user of `ReadVersion`, it creates PlainStateReader just to call this function